src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,25 @@
+package calculator
+
+enum class Operator(
+    private val symbol: String,
+    val operate: (Int, Int) -> Int,
+) {
+    ADD("+", { operand1, operand2 -> operand1 + operand2 }),
+    SUBTRACT("-", { operand1, operand2 -> operand1 - operand2 }),
+    MULTIPLY("*", { operand1, operand2 -> operand1 * operand2 }),
+    DIVIDE("/", { operand1, operand2 ->
+        require(operand2 != 0) { "0으로 나눌 수 없습니다." }
+        operand1 / operand2
+    }),
+    ;
+
+    companion object {
+        fun of(symbol: String): Operator {
+            return values().filter { it.symbol == symbol }
+                .run {
+                    require(isNotEmpty()) { "사칙연산 기호가 올바르지 않습니다. : $symbol" }
+                    first()
+                }
```

Comment:
아래와 같이 `find` 함수를 사용하는 코드로 변경해보면 어떨까요? 
```suggestion
            return values().find { it.symbol == symbol } 
                ?: throw IllegalArgumentException("사칙연산 기호가 올바르지 않습니다. : $symbol")
```

Reply:
유용한 메소드네요
적용했습니다 감사합니다 😀